source/gui/settingsDialogs.py

```diff
@@ -1353,13 +1353,20 @@ def makeSettings(self, settingsSizer):
 		self.audioDetectBrightnessCheckBox=sHelper.addItem(wx.CheckBox(self,label=audioDetectBrightnessText))
 		self.audioDetectBrightnessCheckBox.SetValue(config.conf["mouse"]["audioCoordinates_detectBrightness"])
 
+		# Translators: This is the label for a checkbox in the
+		# mouse settings panel.
+		handleInjectedMouseControlText = _("Handle mouse control from other &applications")

```

Comment:
I don't think the naming of this setting conveys what it does. Maybe something like "Ignore mouse movement triggered by other applications"

Reply:
I started with the word movement, but the problem with that word is that strictly spoken, mouse clicks aren't movement either.

Reply:
Hmm, ignore mouse input would turn the boolean upside down and causes a symmetrical conflict with handle keys from other applications  in the keyboard settings.

How about "Process mouse input from other applications"

Reply:
After considering this a little more, I think that I agree now with the wording using "ignore" now. May be it makes sense to reword the keyboard settings option as well?